docs/tutorial/index.md

```diff
@@ -321,22 +321,22 @@ Time to install UP4W! Click on [this link](https://www.microsoft.com/store/produ
 ![Install Ubuntu Pro for WSL from the Store](./assets/store.png)
 
 
-Once the installation is complete, you will see a **Start** button. Click it. That will start the UP4W Windows agent. When the progress bar indicates that the process has been completed, close the application: At this point your UP4W GUI as well as your UP4W Windows Agent are both up and running, and you can continue to interact with the UP4W Windows Agent via the UP4W GUI, but here we will do that through the Windows registry -- that way you get a taste of what this might look like when you use UP4W to do things at scale.
+Once the installation is complete, you will see a **Start** button. Click it. That will start the UP4W Windows application. We'll use it to configure UP4W at next.
 
 
-### Configure UP4W for Ubuntu Pro and Landscape
-
+```{note}
+Instead of using the GUI, it's possible to configure UP4W through the Windows registry, which enables you to do things at scale.
+```
 
-In the Windows Start Menu, locate the Registry Editor and open it.
+### Configure UP4W for Ubuntu Pro and Landscape
 
-In the left pane, navigate to `HKEY_CURRENT_USER\Software\`.
+In the UP4W GUI click in the label "Already have a token?" to expand the Ubuntu Pro token input field.
 
-Create a new key named `Canonical` and, inside of it, create another key named `UbuntuPro`.
+![UP4W GUI main screen](./assets/up4w_gui.png)
 
-Inside the newly created `UbuntuPro` key, add a string value named `UbuntuProToken` and, in its value data field, paste your Ubuntu Pro token. (The token you retrieved from your Ubuntu Pro dashboard during [Setup](ref::ensure-ubuntu-pro)).
+Paste your token retrieved from your Ubuntu Pro dashboard during [Setup](ref::ensure-ubuntu-pro) and click on the "Confirm" button (the button becomes green when there is a valid token in the input field). The app will then show the Landscape configuration screen.
```

Comment:
I know this is not new, but the way I did most of these labels was (folder::name), so here it'd be `tut::ensure-ubuntu-pro`, as `ref` was for links to the reference.

No need to change it, but just so you know 😅

Reply:
I like that suggestion.